### ISSUE TITLE
Forest Carbon database (ForC-db)

### ISSUE THREAD

**Issue by aornugent** [moja-global/Land_Sector_Datasets#3]
Is this repo good place to note available datasets for analysis? The Smithsonian have an open data base here: https://forc-db.github.io/

**Comment by gmajan** [comment]
Yes it is a good place. I think that @GeoffRoberts and @HarrisLawYF are working on this at the moment and they might want to include this set too. Geoff and Harry can you pls confirm? thx
